app/components/ManageSpecies/index.js

```diff
@@ -118,30 +129,48 @@ const ManageSpecies = () => {
         if (selectedSpecies.length === 0) {
           onPressBack();
         }
-        // setIsLoaderShow(true);
         let species = [...selectedSpecies];
+        let alreadyPresentSpecies = [];
         for (let specie of species) {
-          let scientificName = specie.scientific_name;
-          let speciesId = specie.guid;
-          console.log(scientificName, speciesId, 'Specie');
-          AddUserSpecies(scientificName, speciesId)
+          let currentSpecie;
+          for (let item of specieList) {
+            if (specie.guid === item.speciesId){
+              currentSpecie = item.scientificName;
+              alreadyPresentSpecies.push(currentSpecie);
+              console.log(alreadyPresentSpecies,'already Present');
+              break;
+            } else {
+              currentSpecie = null;
+            }
+          }
+          if (!currentSpecie){
+            AddUserSpecies(specie.scientific_name, specie.guid)
             .then((data) => {
               setSpecieId(data)(speciesDispatch);
               onPressBack();
-              // setIsLoaderShow(false);
             })
             .catch((err) => {
-              // setIsLoaderShow(false);
               console.log(err);
               Alert.alert(
                 i18next.t('label.select_species_error'),
                 i18next.t('label.select_species_enter_valid_input', {
-                  scientificName: specie.scientificName
+                  scientific_name: specie.scientific_name
                 }),
                 [{ text: i18next.t('label.select_species_ok'), onPress: () => console.log('OK Pressed') }],
                 { cancelable: false },
               );
             });
+          }
+        }
+        if (alreadyPresentSpecies.length > 0) {
+          Alert.alert(
+            "Error",
+            `You have added ${alreadyPresentSpecies} already`,
+            [
+              { text: "OK", onPress: () => console.log("OK Pressed") }
+            ],
+            { cancelable: false }
```

Comment:
Please localize this as e.g. https://github.com/Plant-for-the-Planet-org/treemapper/blob/1678849a4ae3e9b2736ed33f6ae5193f86c1a0a5/app/components/AddSpecies/index.js#L100-L107